application-archetype/src/main/resources/archetype-resources/README.md

```diff
@@ -0,0 +1,22 @@
+# ${pluginName} Application Plugin
+
+A custom Application plugin for the Curity Identity Server.
+
+## Building the Plugin
+
+You can build the plugin by issuing the command ``mvn package``. This will produce a JAR file in the ``target`` directory,
+which can be installed.
+
+## Installing the Plugin
+
+To install the plugin, copy the compiled JAR (and all of its dependencies) into the :file:`${IDSVR_HOME}/usr/share/plugins/${pluginName}`
```

Comment:
Triggered from another review: I would repeat from where the user should copy the files:

```suggestion
To install the plugin, copy the compiled JAR (and all of its dependencies)  from the ``target`` directory into the :file:`${IDSVR_HOME}/usr/share/plugins/${pluginName}`
```

Reply:
Applies for other READMEs as well.

Reply:
I changed that in all READMES in the second PR.